### PR TITLE
Implemented deployment using kubernetes

### PR DESCRIPTION
**Pull request by AliAyub007**
Fixes issue #30 

Changes: 
Deployed SUSI Viber bot to Google container engine using kubernetes.

### CODE REVIEW

**Comment by AliAyub007** [comment]
@niranjan94 @chiragw15 @uday96 Kindly review this PR.

File: kubernetes/yamls/lego/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+metadata:
+  name: kube-lego
+  namespace: kube-lego
+data:
+  # modify this to specify your address
+  lego.email: "aliayubkhan007@gmail.com"
```
Comment (niranjan94) [inline kubernetes/yamls/lego/configmap.yaml]: Don't add your email here like this. Just add a dummy email

Comment (AliAyub007) [inline kubernetes/yamls/lego/configmap.yaml]: Yes, I will add documentation soon and sure adding dummy email.

**Comment by AliAyub007** [comment]
@niranjan94 Added documentation now.

**Pull request merged**